src/utilities/rangeFunction.js

```diff
@@ -0,0 +1,13 @@
+const rangeFunction = (arrOfData, range) => {
+  if (range === 3)
```

Comment:
Use dynamic code if possible writing `if` `else` again and again is not a good practice .

Reply:
resolved via voice call